### PR TITLE
Add metric - subscribers per language

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: test/chw.test.js [hunk 1]
```diff
@@ -1470,6 +1470,7 @@ describe("app", function() {
                             assert.deepEqual(metrics['test.chw.percent_complete_registrations'].values, [75]);
                             assert.deepEqual(metrics['test.chw.sum.json_to_jembi_success'].values, [1]);
                             assert.deepEqual(metrics['test.sum.subscriptions'].values, [1]);
+                            assert.deepEqual(metrics['test.sum.subscribers.en'].values, [1]);
```
Comment (gsvr) [inline test/chw.test.js:1478]: @imsickofmaps Perhaps this naming isn't the best considering the line above it.

File: test/personal.test.js [hunk 2]
```diff
@@ -1343,7 +1343,7 @@ describe("app", function() {
         });
 
         describe("after the user enters their birth day", function() {
-            it("should save birth day, thank them and exit", function() {
+            it.only("should save birth day, thank them and exit", function() {
```
Comment (imsickofmaps) [inline test/personal.test.js]: stray `.only`

**Comment by imsickofmaps** [comment]
:+1: if travis happy when the `.only` removed

**Pull request merged**